from hate.logger import logging
from hate.exception import CustomException


logging.info('Welcome to our project')

try:
    a=7